db/migrate/20220114195219_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :postscounter
```

Comment:
As best practices, let's adhere to ruby/rails naming conventions. `:postscounter` should be in snake case i.e. `:posts_counter.` Kindly make this change. 👍

Reply:
Done. Thank you @bimbolabuari